cmd/package_test.go

```diff
@@ -203,3 +203,69 @@ func (s *suite) TestDescribePackage(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(descr, MatchesMultiline, fmt.Sprintf(".*PACKAGE DOCUMENTATION\n%s\n", DefaultText))
 }
+
+func (s *suite) TestRecursiveRunYamls(c *C) {
+	// Prepare.
+	s.importFakeOSvBootstrapPkg(c)
```

Comment:
This is correct. But now that you've mentioned I realized how annoying such implicit assumptions are so I've changed the test a bit so that it now introduces a new package named `fake.demo` that contains the `meta/run.yaml`. I had to leave the logic for `osv.bootstrap` there so that Capstan doesn't fail as it's implicitly required, but I don't abuse it in tests anymore.